### PR TITLE
[12.0][FIX] contract_sale_invoicing: Fixed Issue #440 (when calling Scheduled Actions)

### PR DESCRIPTION
**Pull request by nikul-serpentcs**
Issue: #440

### CODE REVIEW

File: contract_sale_invoicing/models/contract.py [hunk 1]
```diff
@@ -16,16 +16,17 @@ class ContractContract(models.Model):
...
+                ('analytic_account_id', '=', rec.group_id.id),
+                ('partner_invoice_id', 'child_of',
+                 rec.partner_id.commercial_partner_id.ids),
+                ('invoice_status', '=', 'to invoice'),
+                ('date_order', '<=',
+                 '{} 23:59:59'.format(rec.recurring_next_date)),
+            ])
+            if sales:
+                invoice_ids = sales.action_invoice_create()
+                invoices |= self.env['account.invoice'].browse(invoice_ids)[:1]
```
Comment (sbejaoui) [inline contract_sale_invoicing/models/contract.py:32]:
```suggestion
                invoices |= self.env['account.invoice'].browse(invoice_ids)[:1]
        return invoices
```

**Review by sbejaoui — CHANGES_REQUESTED** [review]

**Comment by sbejaoui** [comment]
Can you add a unit test with multiple contract to assure the non-regression 

Thank you

**Comment by nikul-serpentcs** [comment]
> Can you add a unit test with multiple contract to assure the non-regression
> 
> Thank you

@sbejaoui Now Done with Multiple contract

**Comment by nikul-serpentcs** [comment]
@sbejaoui @max3903 Any Update?

File: contract_sale_invoicing/tests/test_contract_sale_invoicing.py [hunk 2]
```diff
@@ -45,3 +45,10 @@ def test_contract_sale_invoicing_without(self):
         self.sale_order.action_confirm()
         self.contract.recurring_create_invoice()
         self.assertEqual(self.sale_order.invoice_status, 'to invoice')
+
+    def test_multi_contract_recurring_create_invoice(self):
+        self.contract.copy()
+        self.contract.copy()
+        self.contract.copy()
+        self.contract.copy()
+        self.env['contract.contract'].cron_recurring_create_invoice()
```
Comment (sbejaoui) [inline contract_sale_invoicing/tests/test_contract_sale_invoicing.py:54]: make an assert here

Comment (nikul-serpentcs) [reply contract_sale_invoicing/tests/test_contract_sale_invoicing.py:54]: But Which values, I called only Scheduled Actions(cron_recurring_create_invoice) not any return here

Comment (sbejaoui) [reply contract_sale_invoicing/tests/test_contract_sale_invoicing.py:54]: check that the cron create correctly the invoices for all contracts

**Comment by nikul-serpentcs** [comment]
@pedrobaeza @max3903 @sbejaoui Now no need to this PR
Because we Already Solved this Issue in https://github.com/OCA/contract/pull/437/files#diff-7387a665f322583d0d8b082a7b7e2cf3R39

 So Please check and close this PR & Issue: #440

**Comment by nikul-serpentcs** [comment]
Fixed Issue in #437

**Pull request closed without merging**